packages/mui-joy/src/Radio/radioClasses.ts

```diff
@@ -1,45 +1,47 @@
 import { generateUtilityClass, generateUtilityClasses } from '../className';
 
 export interface RadioClasses {
-  /** Styles applied to the root element. */
+  /** Classname applied to the root element. */
```

Comment:
```suggestion
  /** Class name applied to the root element. */
```
+ all the lines below